docs/hydra/index.md

```diff
@@ -16,160 +16,126 @@ are.
-a memo in the [Request for Comments](https://www.ietf.org/rfc.html) document
-series published by the IETF Internet Engineering Task Force (IETF). Memos in
-the Requests for Comments (RFC) document series contain technical and
-organizational notes about the Internet. They cover many aspects of computer
-networking, including protocols, procedures, programs, and concepts [...].
-
-The OAuth 2.0 authorization framework enables a third-party application to
-obtain limited access to an HTTP service, either on behalf of a resource owner
-by orchestrating an approval interaction between the resource owner and the HTTP
-service, or by allowing the third-party application to obtain access on its own
-behalf.
-
-In the traditional client-server authentication model, the client requests an
-access-restricted resource (protected resource) on the server by authenticating
-with the server using the resource owner's credentials. In order to provide
-third-party applications access to restricted resources, the resource owner
-shares its credentials with the third party. This creates several problems and
-limitations.
-
-OAuth addresses these issues by introducing an authorization layer and
-separating the role of the client from that of the resource owner. In OAuth, the
-client requests access to resources controlled by the resource owner and hosted
-by the resource server, and is issued a different set of credentials than those
-of the resource owner.
-
-Instead of using the resource owner's credentials to access protected resources,
-the client obtains an access token -- a string denoting a specific scope,
-lifetime, and other access attributes. Access tokens are issued to third-party
-clients by an authorization server with the approval of the resource owner. The
-client uses the access token to access the protected resources hosted by the
-resource server.
-
-Source: [IETF RFC 6749](https://tools.ietf.org/html/rfc6749)
-
-### OAuth 2.0 Example
-
-An end-user (resource owner) can grant a printing service (client) access to her
-protected photos stored at a photo- sharing service (resource server), without
-sharing her username and password with the printing service. Instead, she
-authenticates directly with a server trusted by the photo-sharing service
-(authorization server), which issues the printing service delegation- specific
-credentials (access token).
-
-Source: [IETF RFC 6749](https://tools.ietf.org/html/rfc6749)
+specified in [IETF RFC 6749](https://tools.ietf.org/html/rfc6749). OAuth 2.0
+enables a third-party application to obtain limited access to resources on an
+HTTP server on behalf of the owner of those resources.
+
+Why is this important? Without OAuth 2.0, a resource owner who wants to share
+resources in their account with a third party would have to share their
+credentials with that third party. As an example, let's say you (a resource
+owner) have some photos (resources) stored on a social network (the resource
```

Comment:
This example is super helpful! I changed the "photo sharing service" in it to "social network". 

- I found that "photo sharing service" and "printing service" sound too alike. I kept confusing the two. In the real world, these two functionalities (sharing and printing photos) often go together. On the other hand, it seems more plausible that a social network has lots of photos but not necessarily a way to print them directly.
- We want to provide an example for other information (besides photos) that isn't included in the OAuth scopes. That's hard for a service that only shares photos.

Reply:
Awesome!